doc/lsst.display.firefly/getting-started-lsp.rst

```diff
@@ -34,9 +34,9 @@ Retrieve a simulated LSST image.
 .. code-block:: py
     :name: getting-started-getimage
 
-    from lsst.daf.persistence import Butler
+    from lsst.daf.butler import Butler
     butler = Butler('/project/shared/data/Twinkles_subset/output_data_v2')
```

Comment:
I'm not sure if this is/was a Gen3 butler, but defining a REPO and a COLLECTION would be more illustrative.

Reply:
I changed it to use collection and also realized that the butler.get needed to be fixed.

Reply:
I missed the `butler.get` previously. Good catch.